src/components/layouts/collection-page-layout.tsx

```diff
@@ -388,26 +390,39 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                 {toggle_favorite_url ? (
                   <button
                     onClick={() => {
```

Comment:
I think we should DRY this component up, might come handy for saving other resources.